molecule/default/molecule.yml

```diff
@@ -3,20 +3,31 @@ dependency:
   name: galaxy
 driver:
   name: docker
-lint:
-  name: yamllint
+lint: |
+  yamllint .
```

Comment:
Wonder if the lint step will fail without us specifying `set -e` as the first line.